### DISCUSSION TITLE
Interlaced HDMI Video Output

### DISCUSSION THREAD

**Discussion by edisionnano** [NVIDIA/open-gpu-kernel-modules#625] [Ideas]
Recently I updated my 1060 6gb to a 1660ti on my PC which is hooked to a CRT using a passive HDMI to VGA adapter. On the 1060 I could output interlaced video from the HDMI ports and get 1600x1200@144hz but on the 1660ti interlacing doesn't seem to work.

**Comment by aaronp24** [comment]
Volta and newer don't support interlaced modes.

(Also, there's no such thing as a passive HDMI to VGA adapter)
